modules/builds-strategy-docker-build.adoc

```diff
@@ -6,4 +6,9 @@
 [id="builds-strategy-docker-build_{context}"]
 = Docker build
 
-The docker build strategy invokes the docker build command, and it expects a repository with a Dockerfile and all required artifacts in it to produce a runnable image.
+{product-title} uses Buildah to build a container image from a Dockerfile. For more information on building container images with Dockerfiles, see link:https://docs.docker.com/engine/reference/builder/[the Dockerfile reference documentation].
+
+[TIP]
+====
+If you are working with build arguments, consider reviewing link:https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact[Understand how ARG and FROM interact] in the Dockerfile reference documentation.
```

Comment:
Suggest: If you work... review the link:https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact[Understand how ARG and FROM interact] section...